src/navability/entities/NavAbilityClient.jl

```diff
@@ -24,9 +24,11 @@ end
 function NavAbilityHttpsClient(apiUrl::String="https://api.d1.navability.io")::NavAbilityClient
     dianaClient = GraphQLClient(apiUrl)
     function query(options::QueryOptions)
+        # TODO does Diana or Deloittes return async?
```

Comment:
Looks like the Deloitte one is synchronous as well - https://github.com/DeloitteDigitalAPAC/GraphQLClient.jl, so we'd need to do the async wrapping here

Reply:
Okay, I'll add that in a next PR.